umbraco-cloud/set-up/project-settings/umbraco-cicd/Troubleshooting.md

```diff
@@ -1,5 +1,62 @@
 # Troubleshooting
 
+## Upload Errors
+
+### Failed to read the request form. Multipart body length limit 134217728 exceeded
+
+We currently have a size limit set to 134217728 bytes or about ~128 MB. 
+
+Make sure that the package you are trying to upload does not contain anything unnessary.
+
+Our samples contains examples of how you could zip your repository before uploading it:
+
+{% tabs %}
+{% tab title="Github" %}
+```yaml
+## STEP FROM THE GITHUB YAML SAMPLE
+# zip everything, except what is defined in the '.zipignore'
+  - name: Zip Source Code
+    run: zip -r sources.zip . -x@.zipignore
+    shell: bash
+```
+{% endtab %}
+
+{% tab title="Azure Devops" %}
+```yaml
+## STEP FROM THE AZURE DEVOPS YAML SAMPLE
+# zip everything, except what is defined in the '.zipignore'
+  - script: zip -r dist/sources_$(Build.BuildNumber).zip . -x@.zipignore
+    displayName: Zip source files to package artifact
+```
+{% endtab %}
+
+{% tab title=".zipignore" %}
```

Comment:
@jesp209i is this .zipignore info/steps necessary to be read/done for both Github and Devops? 

Because in this case it might be better to keep this separate than tabs. As for tabs you need to choose like the options like this: https://docs.umbraco.com/umbraco-cms/extending/packages/installing-and-uninstalling-packages#installing-packages so it seems to be suited for devops and github info like above. But for the gitignore if it is important info/steps to be done and if it applied to both, then they should not be put in the same tabs. An idea could be to use dropdowns such as: https://docs.umbraco.com/umbraco-forms/release-notes#release-history or just leave the content as it is outside the tab option. Let me know what you think! 😊

Reply:
@alina-tincas thanks for feedback :
the zip ignore is not steps but rather the actual content of a file. We include it in the script zips for each pipeline, but maybe it makes sense to include it in the actual articles for each pipeline :)